README.md

```diff
@@ -2,35 +2,65 @@
 
-## How to collaborate
+The most recent version of the dapp is available at http://dapp.democracy.earth. For desktop usage, you will need Metamask and a Metamask-supported browser to be able to interact with it. For mobile usage, use a dapp browser / wallet like [Toshi](https://www.toshi.org/).
 
-PRs and feedback are welcomed. A good place to start is to review the current [open issues](https://github.com/DemocracyEarth/contracts/issues), please feel free to engage with us by commenting in a specific issue. Remember you can also check out the [Waffle board](https://waffle.io/DemocracyEarth/contracts) to understand what's being worked on now and what is coming up next.
+## Contributing
 
-## How to test locally
+PRs and feedback are welcomed. A good place to start is to review the current [open issues](https://github.com/DemocracyEarth/dapp/issues), please feel free to engage with us by commenting in a specific issue. Remember you can also check out the [Waffle board](https://waffle.io/DemocracyEarth/contracts) to understand what's being worked on now and what is coming up next.
 
-Project is being developed with NodeJs and Truffle, please make sure you have the latest version of npm before continuing. First follow the instructions to install [Truffle](http://truffleframework.com/):
+Dapp development is discussed actively in the slack channel [#developers](https://democracyearth.slack.com/messages/C5KCH0PD1) in democracyearth.slack.com 
 
-```sh
-$ npm install -g truffle
-```
+The dapp consists of mailnly two elements, contracts and frontend, located respectively in the following folders:
+- **contracts**: Solidity contracts, to be deployed to a local or remote blockchain.
+- **dApp-Srv-Exp**: JS frontend built using npm, bower and http-server
 
-Then clone or fork the repo, do an `npm install`, and you can start running the tests. You will need to run `truffle develop` to start a testing blockchain and then `test` to execute testing files. Please read the [Truffle docs](http://truffleframework.com/docs/getting_started/testing) to learn more about the tool.
+## Development dependencies
 
-## Compile contracts
+- NodeJS (make sure you have the latest version of npm, using `npm i -g npm`)
+  - Bower (`npm install -g bower`)
+  - Truffle (`npm install -g truffle`)
+  - http-server (`npm install -g http-server`)
+- Ganache ([http://truffleframework.com/ganache/]())
```

Comment:
Ganache is optional

Reply:
What do you use to be able to have a virtual blockchain and connect from Metamask? truffle develop? It didn't work for me some time ago because some bugs in Truffle and the default port in Metamask being 8545 instead of truffle's 9545

Reply:
Curious here too. I had been using `truffle develop` for some time now. I tested this with Ganache yesterday just to double check instructions. It should work either way though